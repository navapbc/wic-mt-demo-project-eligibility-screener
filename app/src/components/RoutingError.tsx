import { ReactElement } from 'react'

import Alert from '@components/Alert'

export const RoutingError = (): ReactElement => {
  return (
    <div className="margin-bottom-3">
      <Alert alertBody="routingError" type="error" icon={true} />
    </div>
  )
}

export default RoutingError
