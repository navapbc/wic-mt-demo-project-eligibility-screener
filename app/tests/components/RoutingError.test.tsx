import RoutingError from '@components/RoutingError'

import { testSnapshot } from '../helpers/sharedTests'

it('should match snapshot', () => {
  testSnapshot(<RoutingError />)
})
