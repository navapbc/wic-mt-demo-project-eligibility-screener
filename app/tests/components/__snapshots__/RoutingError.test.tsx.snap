// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div
  className="margin-bottom-3"
>
  <div
    className="usa-alert usa-alert--error "
    role="alert"
  >
    <div
      className="usa-alert__body"
    >
      <p
        className="usa-alert__text"
      >
        Some required responses are missing. Please go back through the form and answer all required questions.
      </p>
    </div>
  </div>
</div>
`;
